helm/charts/determined/templates/change-password.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
```

Comment:
We shuold probably add a:
```
{{- if .Values.defaultPassword }}
```
at the top of this file so this job is created only when the password is being changed.

Reply:
I was searching for this type of functionality for like half of yesterday...